### DISCUSSION TITLE
Current limiting resistors on power to sensors

### DISCUSSION THREAD

**Discussion by aconbere** [srozum/film_camera_tester#41] [Ideas]
Mucking about trying to figure out why my tester isn't working (see #40) I briefly shorted VCC and GND on my sensor and burnt out the nano's voltage regulator.

I think it would be relatively easy to limit the current to the sensors with a resistor. I don't know what the maximum safe draw is for the voltage regulator but I don't think there's an expectation that the sensors will need that much current.

Could just avoid some simple mishaps in the future.

**Comment by Surreal62** [comment]
Mistakes happen! Someone I know (me) connected a rather expensive inertial measurement unit I had repaired that required a maximum 5.5V supply to a test rig power supply without checking it's set voltage. Apparently, the IMU wasn't happy to be hit with 27V... 😬
Both my error for not checking and that of the previous tech for using the supply at all, then setting it to 27V and not returning it to 5.5V
Thankfully, I didn't put the whole batch of twelve units through at once!
